src/boot/ws.js

```diff
@@ -128,13 +128,37 @@ onWebSocket(dbEnum.wsCategory.sessionCreationError, (data) => {
   </center>`
   Notify.create({
     message: html,
-    color: 'negative',
+    color: 'black',
```

Comment:
why are we changing the color?

Reply:
no reason besides I thought it looked nice